src/conffile.c

```diff
@@ -1383,7 +1383,11 @@ int n3n_config_setup_sessiondir (n2n_edge_conf_t *conf) {
 
 
 #ifndef _WIN32
+#ifndef BUILD_OPENWRT
     char *basedir = "/run/n3n";
+#else
+    char *basedir = "/tmp/run/n3n";
```

Comment:
The "/tmp/run" is an implementation detail - the well defined dir to use is "/var/run"